src/controllers/resources.js

```diff
@@ -590,7 +598,7 @@ export default function () {
                     // fix var type and field name
                     const field = head.replace(/\w+\(([^)]+)\)/i, '$1')
                     row[field] = head.startsWith('Number') ? Number(row[head])
-                      : head.startsWith('Boolean') ? Boolean(row[head]) : row[head]
+                      : head.startsWith('Boolean') ? Boolean(isTrue(row[head])) : row[head]
```

Comment:
Boa, mas não precisava criar uma função separado porque você só usa aqui, certo? Além disto, como está o valor `'true'` ou `true` vai dar `false`, acho que assim fica melhor ó:

```suggestion
                      : head.startsWith('Boolean')
                        ? typeof row[head] === 'string' ? row[head].toUpperCase().indexOf('TRUE') > -1
                          : Boolean(row[head])
                        : row[head]
```

Reply:
Sim, vem como 'TRUE' ou 'FALSE', mas acho que assim ficar melhor mesmo